src/pages/beta/[viewType]/curriculum/index.tsx

```diff
@@ -219,9 +238,41 @@ export const getStaticProps: GetStaticProps<
 > = async () => {
   const data = await fetchSubjectPhasePickerData();
 
+  const blogs = await CMSClient.blogPosts({
+    previewMode: true,
+  });
+
+  const serializedPost = blogs
+    .filter((blog) => {
+      return (
+        blog.slug === "how-to-design-a-subject-curriculum" ||
+        blog.slug === "how-to-refresh-your-curriculum-using-oak-units" ||
+        blog.slug === "how-to-design-a-unit-of-study"
+      );
+    })
+    .map(serializeDate);
+
+  // TEMPORARY (HACKY) REARRANGEMENT FOR BLOGS ORDER
```

Comment:
what's the longer term plan for this?

Reply:
At some point will be blogs filtered by curriculum, but currently it's those specific blogs in that specific order

Reply:
AFAIK, the intention is to CMS-ify this entire page, so for this component in particular a content editor would select specific blog posts to be shown in these three slots.